docs/DeveloperGuide.md

```diff
@@ -61,6 +61,81 @@ When the user launches the app, the main program will initialize a `TopicsStorag
 
 ![TopicsStorage load](./images/TopicsStorage_load.png)
 
+###Implementation of commands relating to TopicList, namely:
+
+1. Listing topics in TopicList
+2. Finding a topic in TopicList
+3. Adding a new note
+4. Deleting an existing note
+5. Listing out all notes in a topic
+
+![TopicList_Class_Diagram](./images/TopicListAndNotes.png)
+
+TopicList is an ArrayList of type Displayable, which is one of two interfaces implemented 
+in the code for EDuke8. As such, many of the commands that manipulate the TopicList make 
+use of the package java.util.ArrayList. The TopicList is used to store Topics.
+
+####1) Listing topics in TopicList:
+
+![TopicListSampleSequence](./images/TopicListSampleSequence.png)
```

Comment:
Might want to split this as it may be confusing. Also not sure if you should be returning control to each class instead?